runtime/doc/api.txt

```diff
@@ -3108,6 +3108,130 @@ nvim_tabpage_set_var({tabpage}, {name}, {value})
                     {value}    Variable value
 
 
+==============================================================================
+Autocmd Functions                                                *api-autocmd*
+
+nvim_create_augroup({name}, {*opts})                   *nvim_create_augroup()*
+                Create or get an augroup.
+
+                To get an existing augroup ID, do: >
+                    local id = vim.api.nvim_create_augroup(name, {
+                        clear = false
+                    })
+<
+
+                Parameters: ~
+                    {name}  String: The name of the augroup to create
+                    {opts}  Parameters
+                            • clear (bool): Whether to clear existing commands
+                              or not. Defaults to true. See |autocmd-groups|
+
+                Return: ~
+                    opaque value to use with nvim_del_augroup_by_id
+
+nvim_create_autocmd({event}, {*opts})                  *nvim_create_autocmd()*
+                Create an autocmd.
+
+                Examples:
+                • event: "pat1,pat2,pat3",
+                • event: "pat1"
+                • event: { "pat1" }
+                • event: { "pat1", "pat2", "pat3" }
+
+                Parameters: ~
+                    {event}  The event or events to register this autocmd
+                             Required keys: event: string | ArrayOf(string)
+
+                Parameters: ~
+                    {opts}  Optional Parameters:
+                            • callback: (string|function)
+                              • (string): The name of the viml function to
+                                execute when triggering this autocmd
+                              • (function): The lua function to execute when
+                                triggering this autocmd
+                              • NOTE: Cannot be used with {command}
+
+                            • command: (string) command
+                              • vimscript command
+                              • NOTE: Cannot be used with {callback} Eg.
+                                command = "let g:value_set = v:true"
```

Comment:
here it looks like that command is an example for a collision with callback

Reply:
No, it's not.